articles/specs/launch.0.1.1.xsd

```diff
@@ -475,7 +477,7 @@
           </xs:documentation>
         </xs:annotation>
       </xs:attribute>
-      <xs:assert test="(@name and not(@from)) or (not(@name) and @from)"/>
+      <!-- <xs:assert test="(@name and not(@from)) or (not(@name) and @from)"/> -->
```

Comment:
I guess this isn't supported

Reply:
Oops. This shouldn't be commented out, I only did so for `xmllint` to accept this schema. There're Python modules with full XSD 1.1 support we can use if we ever attempt to validate given launch files prior to execution.